frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -40,6 +41,10 @@ common_settings.screen = {
         require("ui/elements/refresh_menu_table"),
     },
 }
+if Device:isAndroid() then
+    table.insert(common_settings.screen.sub_item_table, require("ui/elements/screen_fullscreen_menu_table"))
```

Comment:
With koreader/koreader-base#493 and koreader/android-luajit-launcher#52 you should be able to test on real Android device. Init android device requires android specific functions like `AConfiguration_getDensity` which do not exist on other platforms.